Content.Server/GameObjects/Components/VendingMachines/VendingMachineComponent.cs

```diff
@@ -146,7 +148,7 @@ private void UserInterfaceOnOnReceiveMessage(ServerBoundUserInterfaceMessage ser
             switch (message)
             {
                 case VendingMachineEjectMessage msg:
-                    TryEject(msg.ID);
+                    TryEject(msg.ID, serverMsg.Session.AttachedEntity!);
```

Comment:
Would be better to handle the null rather than suppress it.

Reply:
Fixed. I'm assuming access denied when null.